DetDescendantReport-images/detdescendantreport.gpr.py

```diff
@@ -2,7 +2,7 @@
     id   = 'det_descendant_report_i',
     name = _('Detailed Descendant Report With All Images'),
     description = _("Produces a detailed descendant report with all images and optional todo list."),
-    version = '1.0.9',
+    version = '1.0.10',
```

Comment:
As far as I understand it, version numbers are incremented automatically when the addon is built for release: [reference this post](https://gramps.discourse.group/t/what-is-the-process-of-merging-pull-requests-in-the-add-ons-project/2400/7?u=codefarmer) by @prculley.

Reply:
jpschewe made a further update and bumped to 1.0.11 .
I was previously told to bump the version when I submit so I did. 

I am a novice github user. Are you saying I should remove the .gpr.py from the PR so that the version number is not updated and let the release process do it instead?

Reply:
@hgohel Let us know how you'd like the versions handled. I'm the author of this report. I have a pending PR that depends on this one and intend to create another PR after that with some more updates.

Reply:
@GaryGriffin, @jpschewe In the discussion I referenced, @prculley says, "Note that the release process bumps up the version number and this results in a commit,..." which _implies_ that version numbers do not need to be manually updated. I don't see guidance in the [Addons Development](https://www.gramps-project.org/wiki/index.php/Addons_development) wiki, so I created a thread asking for clarification on [Gramps Development forum on Discourse](https://gramps.discourse.group/t/should-an-addons-pr-include-version-number-update/2591).

BTW, my review comments are not marked as "change requested" in GitHub, so it should not block this PR from being committed.